app/vmui/packages/vmui/src/pages/ExploreLogs/hooks/useFetchLogs.ts

```diff
@@ -35,6 +35,7 @@ export const useFetchLogs = (server: string, query: string, limit: number) => {
     try {
       return JSON.parse(line);
     } catch (e) {
+      console.error("Failed to parse line to JSON", e);
```

Comment:
This would fire if vlogs returns empty response.
Can we check if `line.length == 0` before parsing?